bindings/python/README.md

```diff
@@ -130,4 +121,757 @@ def main():
+| [addresses]  | <code>list[str]</code> | <code>undefined</code> | The list of output ids to search |
+
+**Returns** the list of [OutputResponse](#outputresponse).
+
+#### get_milestone(index): MilestoneDto
+
+Gets the milestone by the given index.
+
+| Param   | Type             | Default                | Description                |
+| ------- | ---------------- | ---------------------- | -------------------------- |
+| [index] | <code>int</code> | <code>undefined</code> | The index of the milestone |
+
+**Returns** the [MilestoneDto](#milestonedto).
+
+#### get_milestone_utxo_changes(index): MilestoneUTXOChanges
+
+Gets the utxo changes by the given milestone index.
+
+| Param   | Type             | Default                | Description                |
+| ------- | ---------------- | ---------------------- | -------------------------- |
+| [index] | <code>int</code> | <code>undefined</code> | The index of the milestone |
+
+**Returns** the [MilestoneUTXOChanges](#milestoneutxochanges).
+
+### High-Level APIs
+
+#### message(seed (optional), account_index (optional), initial_address_index (optional), inputs (optional), input_range_begin (optional), input_range_end (optional), outputs (optional), dust_allowance_outputs (optional), index (optional), data (optional), data_str (optional), parents (optional)): Message
+
+Build a message.
+
+| Param                    | Type                                 | Default                | Description                                  |
+| ------------------------ | ------------------------------------ | ---------------------- | -------------------------------------------- |
+| [seed]                   | <code>str</code>                     | <code>undefined</code> | The hex-encoded seed of the account to spend |
+| [account_index]          | <code>int</code>                     | <code>undefined</code> | The account index                            |
+| [initial_address_index]  | <code>int</code>                     | <code>undefined</code> | The initial address index                    |
+| [inputs]                 | <code>list[[Input](#input)]</code>   | <code>undefined</code> | Inputs                                       |
+| [input_range_begin]      | <code>int</code>                     | <code>undefined</code> | The begin index of the input                 |
+| [input_range_end]        | <code>int</code>                     | <code>undefined</code> | The end index of the input                   |
+| [outputs]                | <code>list[[Output](#output)]</code> | <code>undefined</code> | Outputs                                      |
+| [dust_allowance_outputs] | <code>list[[Output](#output)]</code> | <code>undefined</code> | Dust allowance output to the transaction     |
+| [index]                  | <code>str</code>                     | <code>undefined</code> | The indexation string                        |
```

Comment:
We also have `index_raw: Option<&[u8]>,` for the indexation as bytes

Reply:
Added. Thank you for pointing this out.